web-ts/tools/webpack.config.ts

```diff
@@ -1,19 +1,16 @@
-import {Configuration, DefinePlugin, LoaderOptionsPlugin, optimize} from "webpack";
-import {join, resolve} from 'path'
-import {CleanWebpackPlugin} from 'clean-webpack-plugin'
-import {readFileSync} from "fs";
-import {BundleAnalyzerPlugin} from 'webpack-bundle-analyzer'
+import { Configuration, DefinePlugin, LoaderOptionsPlugin, optimize } from 'webpack'
+import { join, resolve } from 'path'
+import { CleanWebpackPlugin } from 'clean-webpack-plugin'
+import { readFileSync } from 'fs'
+import { BundleAnalyzerPlugin } from 'webpack-bundle-analyzer'
 import MomentTimezoneDataPlugin from 'moment-timezone-data-webpack-plugin'
 import CopyPlugin from 'copy-webpack-plugin'
 import AssetsPlugin from 'assets-webpack-plugin'
 import HtmlWebpackPlugin from 'html-webpack-plugin'
+import loadBuildConfig, { ANDROID, IOS, WEB } from 'build-configs'
+import MomentLocalesPlugin from 'moment-locales-webpack-plugin'
+import translationsConfig from 'translations/src/config'
```

Comment:
Importing from `translations/src/config` instead of from `translations` fixes a ts cannot resolve module error for .json files. Absolutely no idea why...